api/google_api/routes/sheets.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const { SceGoogleApiHandler } = require('../util/SceGoogleApiHandler');
+const {
+  OK,
+  BAD_REQUEST
+} = require('../../util/constants').STATUS_CODES;
+
+router.post('/addToSpreadsheet', async (req, res) =>{
```

Comment:
add test cases for this similar to [`test/Calendar.js`](https://github.com/SCE-Development/Core-v4/blob/dev/test/Calendar.js)